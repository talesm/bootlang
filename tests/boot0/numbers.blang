writeln(3.add(2).toString());
writeln(3.sub(2).toString());
writeln(3.equals(3).toString());
writeln(3.equals(2).toString());
writeln(3.isAfter(2).toString());
writeln(3.isAfter(3).toString());
writeln(2.isBefore(3).toString());
writeln(3.isBefore(3).toString());
writeln(number.parseOrPanic('9').add(2).toString());
writeln(number.parseOrPanic('42').sub(21).toString());