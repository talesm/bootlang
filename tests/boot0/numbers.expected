5
1
true
false
true
false
true
false
11
21
